### PR TITLE
[Rev 1 Demo] Update checkboxes to improve readability

### LINKED ISSUE #688: [Rev 1 Demo] Shop Lookup - Filter buttons are misleading

### PR DESCRIPTION
**Pull request by TimChoy**
Close #688

### CODE REVIEW

**Comment by arkinmodi** [comment]
Clicking the text `No Filter` for "Part Condition" is will update "Part Type" to "No Filter". It correctly works for all other fields but not "No Filter".

File: src/pages/index.tsx [hunk 1]
```diff
@@ -21,17 +21,31 @@ import ShopLookupSkeleton from "../components/lookup/ShopLookupSkeleton";
...
+  { key: "OEM", value: "OEM Only" },
+  { key: "AFTERMARKET", value: "Aftermarket Only" },
+  { key: "OEM & AFTERMARKET", value: "OEM & Aftermarket" },
+];
+const partConditions = [
+  { key: "No Filter Condition", value: "No Filter" },
+  { key: "NEW", value: "Only New Parts" },
+  { key: "USED", value: "Only Used Parts" },
+  { key: "NEW & USED", value: "New & Used Parts" },
+];
```
Comment (arkinmodi) [inline src/pages/index.tsx:35]: You could mark these as `const`, so you don't need to use `!` everywhere to override the type. E.g., `partTypes[0]!.key` becomes `partTypes[0].key`.

```suggestion
const partTypes = [
  { key: "No Filter Type", value: "No Filter" },
  { key: "OEM", value: "OEM Only" },
  { key: "AFTERMARKET", value: "Aftermarket Only" },
  { key: "OEM & AFTERMARKET", value: "OEM & Aftermarket" },
] as const;
const partConditions = [
  { key: "No Filter Condition", value: "No Filter" },
  { key: "NEW", value: "Only New Parts" },
  { key: "USED", value: "Only Used Parts" },
  { key: "NEW & USED", value: "New & Used Parts" },
] as const;
```

**Review by arkinmodi — COMMENTED** [review]
Clicking the `Reset Filters` button causes all the shops to disappear.

I don't think we should have a "No Filter" option and an option for both, since they do the same thing.

**Comment by TimChoy** [comment]
Closing PR as talking with Leon and Joy and looking at examples, checkboxes still make more sense. It follows conventions of other websites, maybe some phrasing of the options to make it more clear would be better.

**Pull request closed without merging**